src/WinkPost.php

```diff
@@ -50,6 +50,15 @@ class WinkPost extends Model
         'publish_date'
     ];
 
+    /**
+     * The attributes that should be casted.
+     *
+     * @var array
+     */
+    protected $casts = [
+        'meta' => 'array'
```

Comment:
Just asking.

Why is this needed? The column is already json 🤔

Great work in this PR man 🙌

Reply:
@Lloople thanks very much, appreciate it, and the quick attention to this PR 😃 

This cast here is intended mainly for convenience, both on backend and front. Without it, the api response for the meta field looks not like it does in the screenshots above, but more like the below:
![image](https://user-images.githubusercontent.com/143244/48318270-bdde2280-e5f5-11e8-805c-8fa7bd7f88ac.png)
The front end would then need to json decode the meta data before assigning to properties in the views.

Similarly, on the front end each of the fields is set up as `form.meta.NAME`, and posted through the standard api post update code. Without the json field casting on the post model in the backend, this flow throws an error when the store is done, as we're trying an array to single conversion.
The front end could be updated to stringify the `form.meta` data before submission to get around this also (or do something similar in the controller), but by adding the casts value, we get to treat `meta` like an array on the way in and out, without having to write any conversion functions ourselves anywhere else in the code.